### PR TITLE
[TUBEMQ-459] init agent project and add agent common module

### PR DESCRIPTION
**Pull request by yuanboliu**

### CODE REVIEW

File: tubemq-agent/agent-common/src/main/java/org/apache/tubemq/agent/conf/Configuration.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+     */
+    public int getInt(String key) {
+        JsonElement value = configStorage.get(key);
+        if (value == null) {
+            throw new NullPointerException("null value for key " + key);
+        }
+        return value.getAsInt();
+    }
+
+    /**
```
Comment (gosonzhang) [inline tubemq-agent/agent-common/src/main/java/org/apache/tubemq/agent/conf/Configuration.java:166]: Need to translate Chinese into English, the same below

File: tubemq-agent/README.zh.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# 项目说明
```
Comment (gosonzhang) [inline tubemq-agent/README.zh.md:1]: Need to translate Chinese into English

File: tubemq-agent/pom.xml [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+        <oro.version>2.0.8</oro.version>
+        <aviator.version>2.2.1</aviator.version>
+        <avro.version>1.7.2</avro.version>
+
+        <netty.version>3.8.0.Final</netty.version>
+        <snappy.version>1.0.4.1</snappy.version>
+        <protobuf.version>2.5.0</protobuf.version>
+        <httpclient.version>4.5.13</httpclient.version>
+        <fastjson.version>1.2.68</fastjson.version>
+        <sleepycat.version>6.4.9</sleepycat.version>
```
Comment (gosonzhang) [inline tubemq-agent/pom.xml]: The 6.4.9 version of sleepycat is not the Apache v2 version

Please check the LICENSE of the code and the dependent component packages before incorporating the code on a large scale.

**Review by gosonzhang — CHANGES_REQUESTED** [review]
If the source codes have parts that refer to external components, please add an explanation in the LICENSE of the main project and list the other party’s LICENSE authorization category

**Comment by yuanboliu** [comment]
@gosonzhang add new pr to address your comments. Thanks

**Pull request closed without merging**